tests/labeler/chexpert/config_tests.py

```diff
@@ -0,0 +1,56 @@
+import csv
+import os
+import tempfile
+from bllipparser import ModelFetcher
+
+
+class WeatherTestsConfig:
+    def __init__(
+            self,
+            parsing_model_dir: str = ModelFetcher.download_and_install_model(
```

Comment:
I've never seen a function called as standard arg, so while it probably makes sense, I'd make the conclusion it's not good style. I'd also have doubts that unwanted side effects might happen. Rather check for None in the constructor and call the function then